### ISSUE TITLE
Compilation issue 

### ISSUE THREAD

**Issue by GunpowderGuy** [cldershem/iced-rs-website#2]
I am getting the following error : 
"Error: Failed to render content of /home/diego1/Documentos/iced-rs-docs/content/docs/hello_world.md
Reason: Failed to render code shortcode
Reason: Tried to render `shortcodes/code.html` but the template wasn't found
"

**Comment by GunpowderGuy** [comment]
@cldershem

**Comment by cldershem** [comment]
Can you give me a little more info of where you're getting that error? The first thing I see is that it's got the wrong files extension. The shortcode should be `.md` not `.html`. I'm not sure where that error is coming from, so it's a little difficult to troubleshoot.

**Comment by GunpowderGuy** [comment]
I just followed the instructions : cloned the repo, opened the folder and ran zola serve,

**Comment by cldershem** [comment]
Can you tell me your zola version and what OS you're on? I just did a fresh clone and ran it with zola 0.13.0 on Ubuntu and I can't reproduce the issue.

**Comment by GunpowderGuy** [comment]
I am on fedora 33 and i installed the official zola package, so it should be the latest version available on my os. The same issue pops up on my two fedora pcs

**Comment by cldershem** [comment]
What's the output of `zola --version`? Did you install it with `dnf`? Could you maybe try either the snap package, docker, or building it from source with `cargo`?

I searched the `zola` issue tracker and don't see anything that is obviously related. My only guess right this second is that it's a version mismatch. There appears to have been some changes to `zola` and file system paths in between 0.11, 0.12, and 0.13.0. I'm not a Fedora user, but it appears (from https://src.fedoraproject.org/rpms/zola) that it might be version 0.12 in Fedora's repos.

Once we can confirm it's a version mismatch, we can figure out how to get you the right version.

**Comment by GunpowderGuy** [comment]
Yes, i installed the official fedora package using dnf. The current version is  "zola 0.11.0"

**Comment by GunpowderGuy** [comment]
I upgraded early to fedora 34 and that fixed the issue. Its zola version (0.12.0) is able to compile this website and better yet, snap finally works fine with that distro ( docker still seems overly complicated but that could be me ). 

What do you recommend for people that dont have easy access to either package manager or an up to date native zola package ?
